config/features.yml

```diff
@@ -1511,6 +1511,9 @@ features:
     actor_type: user
     description: Implement multiple page response pattern
     enable_in_development: true
+  pension_introduction_update
```

Comment:
Your tests are failing because you're missing a `:` at the end of this feature name.

Reply:
Thank you for catching that